blocks/hero-advanced/hero-advanced.js

```diff
@@ -1,41 +1,36 @@
+import { isVideo, loadScript } from '../../scripts/scripts.js';
 // eslint-disable-next-line object-curly-newline
 import { div, a, span, img } from '../../scripts/dom-helpers.js';
 import { decorateIcons } from '../../scripts/lib-franklin.js';
 import { loadFragment } from '../fragment/fragment.js';
 import { buildHero } from '../hero/hero.js';
-import { isVideo, videoButton } from '../../scripts/scripts.js';
 
 async function openMediaGallery(mg) {
   const overlay = mg.parentElement.querySelector('.overlay');
   if (overlay) overlay.classList.add('open');
 }
 
 async function buildMediaGallery(mg) {
-  // container
-  const overlay = div({ class: 'overlay' });
-  const content = div({ class: 'overlay-content' });
-  const carousel = div({ class: 'overlay-carousel' });
-  mg.after(overlay);
+  const overlay = document.createElement('div');
+  overlay.classList.add('overlay');
+  const content = document.createElement('div');
+  content.classList.add('overlay-content');
+  const carousel = document.createElement('div');
+  carousel.classList.add('overlay-carousel');
```

Comment:
Why did you opt out of using the dom-helpers here? The code before looked cleaner:

```suggestion
  const overlay = div({ class: 'overlay' });
  const content = div({ class: 'overlay-content' });
  const carousel = div({ class: 'overlay-carousel' });
```